depthmapX/treeWindow.h

```diff
@@ -24,28 +24,48 @@ QT_BEGIN_NAMESPACE
 class QEvent;
 class QTreeWidgetItem;
 
-class indexWidget : public QTreeWidget
+class IndexWidget : public QTreeWidget
 {
     Q_OBJECT
 
 public:
-    indexWidget(QWidget *parent = 0, bool custom = true);
-    ~indexWidget();
+    IndexWidget(QWidget *parent = 0);
+    ~IndexWidget();
+
+    QString m_mapColumn = "Map";

```

Comment:
Shouldn't those be const? Especially as they are public and currently some idiot could write code anywhere in dmX to change them on the fly and hillarity would ensue...
Might even be better to turn them in to an enum with values 0 and 1 and use those as index straight away rather than looking up positions in a string list a lot...